plugins/ScheduledReports/Updates/2.14.0-b2.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\ScheduledReports;
+
+use Piwik\Common;
+use Piwik\Db;
+use Piwik\Updater;
+use Piwik\Updates;
+
+/**
+ * Update for version 2.14.0-b2.
+ */
+class Updates_2_14_0_b2 extends Updates
+{
+    public function doUpdate(Updater $updater)
+    {
+        $reportTable = Common::prefixTable('report');
+        $updateSingleReportDescSql = "UPDATE " . $reportTable . " SET description = \"%s\" WHERE idreport = %s";
```

Comment:
Would have to change Updater.php, which uses exec. Could do that in another pr first.